public/js/util.js

```diff
@@ -179,3 +224,282 @@ $.Util.ajaxCall = function(type, url, form) {
 $.Util.toastr = function(type, message) {
     toastr.type(message);
 }
+
+/* setupDashboard()
```

Comment:
Not fussed either way. A lot of the JS here relies on the AJAX functions added but we can include util.js as a base and then specific ones from there.